docs/docs/api/path.md

```diff
@@ -7,8 +7,10 @@ sidebar_label: Path
 `Path` allow a parent view to give an arbitrary path to a child view at runtime.
 
 It differs from rest of the path-composition operators in its usage. Path don't
-go in the header of a producer or view, but is used like state-manipulation
-operators.
+go in the header of a producer or view, but is instead used for accessing values
+in the state.
+
+## Example
 
 For example, imagine a `LoginForm` view, whose producer performs the login, and
```

Comment:
I guess that the argumentation here is that if you want to do a reusable component, and not hardcode paths into that component, you can opt-in on using `Path`.